10_Examples/jett-onboarding/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.8)
+project(jett-onboarding)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(std_msgs REQUIRED)
+
+find_package(ament_cmake_gtest REQUIRED)
+
+# uncomment the following section in order to fill in
+# further dependencies manually.
+# find_package(<dependency> REQUIRED)
+add_executable(listener src/subscriber_member_function.cpp)
+ament_target_dependencies(listener rclcpp std_msgs)
+
+add_executable(talker src/publisher_member_function.cpp)
+ament_target_dependencies(talker rclcpp std_msgs)
+install(TARGETS
+  talker
+  listener
+  DESTINATION lib/${PROJECT_NAME})
```

Comment:
I would either have separate install(TARGETS... for each target (target being an executable or library), or have one big one at the very bottom of the file.

Doesnt actually matter in this case, just for readability. A lot of times we can get away with pattern matching CMake stuff off a working project, but it is good to treat it the same way you would regular code (meaning, make it clean and readable for others/future you).